core.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/impl/locks/CASLockFactoryBuilder.java

```diff
@@ -37,6 +38,7 @@ public class CASLockFactoryBuilder
     private long myCacheExpiryTimeInSeconds = DEFAULT_EXPIRY_TIME_IN_SECONDS;
     private ConsistencyType myConsistencyType = DEFAULT_CONSISTENCY_SERIAL;
     private Node myNode;
+    private UUID myHostId;
```

Comment:
You already have the node, why do you need this?

Reply:
we do not need Node object here. UUID is enough. My plan was to remove Node in clean up patch.